nodes/src/nodes/db_neo4j/IInstance.py

```
@@ -0,0 +1,381 @@
+
+        try:
+            query_json = self._buildCypherQuery(question_text)
+            is_valid = query_json.get('isValid', '').lower() == 'true'
+            cypher = query_json.get('query', '')
+
+            executed = is_valid and bool(cypher) and _is_cypher_safe(cypher)
+
+            if executed:
+                result = self.IGlobal._run_query(cypher)
+            else:
+                result = cypher
```

Ruff (0.15.7)

[warning] Use ternary operator `result = self.IGlobal._run_query(cypher) if executed else cypher` instead of `if`-`else`-block

Replace `if`-`else`-block with `result = self.IGlobal._run_query(cypher) if executed else cypher`

(SIM108)